mappings/net/minecraft/block/NoteBlock.mapping

```diff
@@ -10,3 +10,5 @@ CLASS net/minecraft/unmapped/C_nzuwmjdt net/minecraft/block/NoteBlock
 		ARG 1 entity
 		ARG 3 world
 		ARG 4 pos
+	METHOD m_zettshxq getPitchFromNote (I)F
```

Comment:
a little verbose, why not `getNotePitch` or just `getPitch`?

Reply:
ah, i did "inspire" myself on mojmap :P
will consider those options